ecosystem.config.cjs

```diff
@@ -0,0 +1,13 @@
+module.exports = [
+	{
+		script: 'cli.js',
+		name: 'directus',
+		exec_mode: 'cluster',
+		instances: 0,
```

Comment:
```suggestion
		instances: 'max',
```
Will be more intuitive, since I was about to ask why 0 😅
https://pm2.keymetrics.io/docs/usage/cluster-mode/